README.md

```diff
@@ -1,3 +1,5 @@
+![Logo](.README_images/full_logo.png)
```

Comment:
why you need all the other added images?

Reply:
for safe keeping I guess

Reply:
ok :)